pkg/mount/block/blockdev.go

```diff
@@ -211,20 +210,7 @@ func (b *BlockDev) PCIInfo() (*pci.PCI, error) {
 		return nil, fmt.Errorf("Unable to find backing pci device with device and vendor files for %v", b.Name)
 	}
 
-	// Read both files into the pci struct and return
-	device, err := ioutil.ReadFile(dp)
-	if err != nil {
-		return nil, fmt.Errorf("Error reading device file: %v", err)
-	}
-	vendor, err := ioutil.ReadFile(vp)
-	if err != nil {
-		return nil, fmt.Errorf("Error reading vendor file: %v", err)
-	}
-	return &pci.PCI{
-		Vendor:   strings.TrimSpace(string(vendor)),
-		Device:   strings.TrimSpace(string(device)),
-		FullPath: p,
-	}, nil
+	return pci.OnePCI(dp)
```

Comment:
this doesn't seem right. The equivalent of what was there before would be pci.OnePCI(filepath.Dir(dp)), wouldn't it?

Reply:
nice catch, yeah, it had to be (p)